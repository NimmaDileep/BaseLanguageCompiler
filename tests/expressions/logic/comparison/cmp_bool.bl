print(true < false);
print(true > false);
print(true <= false);
print(true >= false);

print(true <= true);
print(true >= true);
