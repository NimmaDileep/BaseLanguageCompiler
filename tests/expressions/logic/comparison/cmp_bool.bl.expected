false
true
false
true
true
true
